code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -41,6 +41,8 @@ All effects don't start immediately, but rather get worse over time; the rate is
 			booze_power *= 0.7
 		if(HAS_TRAIT(C, TRAIT_LIGHT_DRINKER))
 			booze_power *= 2
+		if(isethereal(C)) //alcohol has no effect on Ethereals
```

Comment:
Change this to a trait, `TRAIT_BOOZE_IMMUNITY` instead?

Reply:
Yeah that sounds better